swap_meet/clothing.py

```diff
@@ -1,2 +1,15 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition=None):
+        if condition == None:
+            super().__init__(category="Clothing")
+        else:
+            super().__init__( condition=condition, category='Clothing')
```

Comment:
Great use of super()!

(Watch out for the extra space)

```suggestion
            super().__init__(condition=condition, category='Clothing')
```